dist/tools/openocd/openocd.sh

```diff
@@ -50,6 +50,11 @@
 #               TELNET_PORT:    port opened for telnet connections
 #               DBG:            debugger client command, default: 'gdb -q'
 #               TUI:            if TUI!=null, the -tui option will be used
+# debug-client: debug-client <elffile>
+#               connects to a running debug-server
+#               GDB_PORT:       port opened for GDB connections
```

Comment:
How can the server be specified to which this connects?

Reply:
`OPENOCD_SERVER_ADDRESS` becomes part of `DBG_FLAGS` in `openocd.sh`.
```sh
: ${DBG_DEFAULT_FLAGS:=-q -ex \"tar ext ${OPENOCD_SERVER_ADDRESS}:$(( GDB_PORT + GDB_PORT_CORE_OFFSET ))\"}
: ${DBG_FLAGS:=${DBG_DEFAULT_FLAGS} ${DBG_EXTRA_FLAGS}}
```

Reply:
Do you want a `GDB_ADDRESS` which is set to `OPENOCD_SERVER_ADDRESS`?

Reply:
Ah no, I see that `OPENOCD_SERVER_ADDRESS` is now documented further up in this file.